Misc/NEWS.d/next/Library/2020-05-02-01-01-30.bpo-29753.n2M-AF.rst

```diff
@@ -0,0 +1 @@
+Fixes packed bitfields population.
```

Comment:
This shows up in the changelog with no other context other than a link to the issue ticket, so you should elaborate as best you can in 1-2 sentences what was broken before and what is now fixed.